### PR TITLE
cloud: refine the scaling a cluster doc

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/scale-tidb-cluster.md [hunk 1]
```diff
@@ -39,35 +88,28 @@ When a cluster is running, you cannot increase its node size. To make such chang
...
     - Current node size: xxx
     - Expected node size: xxx
 
-## Scale in a cluster
-
-To make the following scale-in or scale-down changes, you need to submit a support ticket. We will contact you and complete the scaling within the agreed time.
-
-- Decrease the node size of TiDB, TiKV, or TiFlash
-- Scale in TiKV or TiFlash nodes
+### Decrease node size
```
Comment (SunRunAway) [inline tidb-cloud/scale-tidb-cluster.md:101]: 这个先不写吧。

Comment (qiancai) [reply tidb-cloud/scale-tidb-cluster.md:101]: 那要加一句说明吗？不然看起来以为是文档写漏了这种情况

File: tidb-cloud/scale-tidb-cluster.md [hunk 2]
```diff
@@ -11,22 +11,67 @@ aliases: ['/tidbcloud/beta/scale-tidb-cluter']
...
-To scale a cluster, take the following steps:
+- Node number of TiDB, TiKV, and TiFlash
+- Storage size of TiKV and TiFlash
+- Node size (including vCPUs and memory) of TiDB, TiKV, and TiFlash
 
-1. Navigate to **Active Clusters** page and click the name of a cluster that you want to scale. The overview page of the cluster is displayed.
+For information about how to determine the size of your TiDB cluster, see [Determine Your TiDB Size](/tidb-cloud/size-your-cluster.md).
+
+## Change node number
+
```
Comment (TomShawn) [inline tidb-cloud/scale-tidb-cluster.md:23]: Please add a short description between the two headings.

File: tidb-cloud/scale-tidb-cluster.md [hunk 3]
```diff
@@ -11,22 +11,67 @@ aliases: ['/tidbcloud/beta/scale-tidb-cluter']
...
+To submit a support ticket, perform the steps in [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md). For each node to be scaled, provide the following information in the **Description** box:
+
+- Cluster name: xxx
+- Cloud provider: GCP or AWS
+- Node type: TiKV or TiFlash
+- Current node number: xxx
+- Expected node number: xxx
+
+## Change storage size
+
```
Comment (TomShawn) [inline tidb-cloud/scale-tidb-cluster.md:59]: Please add a short description between the two headings.

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: tidb-cloud/scale-tidb-cluster.md (hunk 2 above)
Comment (qiancai) [reply tidb-cloud/scale-tidb-cluster.md:23]: Thanks for the comment. The short description is added now.

File: tidb-cloud/scale-tidb-cluster.md (hunk 3 above)
Comment (qiancai) [reply tidb-cloud/scale-tidb-cluster.md:61]: Thanks for the comment. The short description is added now.

Comment (TomShawn) [reply tidb-cloud/scale-tidb-cluster.md:61]: LGTM. Thanks~

File: tidb-cloud/scale-tidb-cluster.md (hunk 2 above)
Comment (TomShawn) [reply tidb-cloud/scale-tidb-cluster.md:23]: LGTM. Thanks~

**Comment by TomShawn** [comment]
LGTM

**Pull request merged**